index.html

```diff
@@ -1,72 +1,131 @@
-</head>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./resources/css/styles.css" />
+    <script src="/javascript.js" defer></script>
+  </head>
 
-<body>
+  <body>
     <header class="header">
-        <img src="./resources/images/logoneu.svg" alt="Quiz Time" class="header__img">
+      <img
+        src="./resources/images/logoneu.svg"
+        alt="Quiz Time"
+        class="header__img"
+      />
     </header>
-    <main class="main">
-        <section class="card">
-            <button class="card__bookmark" aria-label="Bookmark"></button>
-            <h2 class="card__question">What is BEM?</h2>
-            <button class="card__button">Show Answer</button>
-            <p class="card__answer card__answer--hidden">Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
-                adipisci maiores magnam! Eos repellendus minus unde eveniet quasi veniam perspiciatis doloremque
-                similique! Et mollitia repellendus hic distinctio minima. Numquam, esse.</p>
-            <ul class="tags">
-                <li class="tags__item">BEM</li>
-                <li class="tags__item">CSS</li>
-                <li class="tags__item">Structure</li>
-            </ul>
-        </section>
-        <section class="card">
-            <button class="card__bookmark--marked" aria-label="Bookmarked"></button>
-            <h2 class="card__question">What is BEM?</h2>
-            <button class="card__button">Show Answer</button>
-            <p class="card__answer card__answer--hidden">Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
-                adipisci maiores magnam! Eos repellendus minus unde eveniet quasi veniam perspiciatis doloremque
-                similique! Et mollitia repellendus hic distinctio minima. Numquam, esse.</p>
-            <ul class="tags">
-                <li class="tags__item">BEM</li>
-                <li class="tags__item">CSS</li>
-                <li class="tags__item">Structure</li>
-            </ul>
-        </section>
-        <section class="card">
-            <button class="card__bookmark" aria-label="Bookmark"></button>
-            <h2 class="card__question">What is BEM?</h2>
-            <button class="card__button">Show Answer</button>
-            <p class="card__answer card__answer--hidden">Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
-                adipisci maiores magnam! Eos repellendus minus unde eveniet quasi veniam perspiciatis doloremque
-                similique! Et mollitia repellendus hic distinctio minima. Numquam, esse.</p>
-            <ul class="tags">
-                <li class="tags__item">BEM</li>
-                <li class="tags__item">CSS</li>
-                <li class="tags__item">Structure</li>
-            </ul>
-        </section>
+    <main class="main main--show" id="main__home">
```

Comment:
I think it would be better if you add a `main--hidden` class instead of `main--show`. It would be easier for other developers to know what is going on in your HTML when they see the class `main--hidden` on three `main` elements instead of `main--show` at just one.